### DISCUSSION TITLE
Typography text secondary color

### DISCUSSION THREAD

**Discussion by Sagie501** [ant-design/ant-design#45412] [Q&A]
Hi!

When I am using Typography text with type secondary like this: `<Typography.Text type="secondary">My Text</Typography.Text>`.

The color of that text is this: `rgba(0,0,0,.45)`.

However, in the token the value of `colorTextSecondary` is `rgba(0, 0, 0, 0.65)` and the color of `colorTextTertiary` is `rgba(0,0,0,.45)`.

I find it a bit confusing that it doesn't aligned with  the Typography component.

I think we might align it and make the Typography.Text secondary color to be `rgba(0, 0, 0, 0.65)` just like in the token.

Furthermore, I think that it will be a good addition to add `tertiary` and `quaternary` types to the `Typography.Text` type component with the relevant colors.

What do you think?

**Comment by Sagie501** [comment]
Any one?

**Comment by YarinDay** [comment]
👍 Also here

**Comment by vandercloak** [comment]
Is it possible the secondary color is calculated of base text color?

<img width="540" alt="image" src="https://github.com/ant-design/ant-design/assets/7164984/d6d2278b-2bae-450e-b54e-f8f2e8f20988">
